### PR TITLE
Documentation/reference/appointment types

### PR DESCRIPTION
**Pull request by TatianaLekhatkova**
Created a separate file about appointment_types

### CODE REVIEW

**Comment by TatianaLekhatkova** [comment]
The latest commit is #d1abf58

File: docs/reference/appointment-types.md [hunk 1]
```diff
@@ -0,0 +1,1192 @@
...
+}
+---
+
+# Appointment types
+
+:::::panel
+
+:::: left
+
+You can use the `/v2/vytemes/:nickname/types` endpoints to manage the availabilities of your users.
```
Comment (saintmac) [inline docs/reference/appointment-types.md:13]: the appointment types of your users

File: docs/reference/appointment-types.md [hunk 2]
```diff
@@ -0,0 +1,1192 @@
...
+
+<attribute name="name" type="string">
+  
+  Name of the Appointment type
+  
+</attribute>
+ 
+<attribute name="handle" type="string">
+  
+Handle is used at the endpoints as a unique identificator of the Appointment type. Created atomatically from the name by urlifying. For example, name `Conference lunch` will be transfomed into `conference-lunch`.
```
Comment (saintmac) [inline docs/reference/appointment-types.md:60]: It's unique for a vyteme page. But several Vyteme pages can have the same 'conference-lunch' appointment type

File: docs/reference/appointment-types.md [hunk 3]
```diff
@@ -0,0 +1,1192 @@
...
+
+Description shown on the Appointment type page.
+
+</attribute>
+
+<attribute name="public" type="boolean">
+
+Whether or not the Appointment type is public.
+
+</attribute>
```
Comment (saintmac) [inline docs/reference/appointment-types.md:86]: undocument this. We're not actually using it.

**Review by saintmac — CHANGES_REQUESTED** [review]

File: docs/reference/appointment-types.md [hunk 4]
```diff
@@ -0,0 +1,1192 @@
...
+
+> RESPONSE SAMPLE
+
+```json light-code
+{
+  "n": 1,
+  "ok": 1
+}
+```
+
```
Comment (saintmac) [inline docs/reference/appointment-types.md:1189]: no the response is actually `204` deleted or `404` not found

**Review by saintmac — CHANGES_REQUESTED** [review]

File: docs/reference/appointment-types.md (hunk 1 above)
Comment (TatianaLekhatkova) [reply docs/reference/appointment-types.md]: done a004104d92e7775d157df1b7c2050653f7e7f587

File: docs/reference/appointment-types.md (hunk 2 above)
Comment (TatianaLekhatkova) [reply docs/reference/appointment-types.md]: done a004104d92e7775d157df1b7c2050653f7e7f587

File: docs/reference/appointment-types.md (hunk 3 above)
Comment (TatianaLekhatkova) [reply docs/reference/appointment-types.md]: done a004104d92e7775d157df1b7c2050653f7e7f587

File: docs/reference/appointment-types.md (hunk 4 above)
Comment (TatianaLekhatkova) [reply docs/reference/appointment-types.md]: done a004104d92e7775d157df1b7c2050653f7e7f587

**Pull request merged**